.gitignore

```diff
@@ -72,7 +72,7 @@
 !/plugins/MauticZapierBundle
 !/plugins/index.html
 
-/themes/*
+
```

Comment:
This should be revert. This change doesn't make sense.

Reply:
Sorry, I really should not be commit.